app/src/main/java/com/safetyheads/akademiaandroida/MainActivity.kt

```diff
@@ -24,6 +25,17 @@ class MainActivity : AppCompatActivity() {
             fragmentTransaction.replace(binding.frameLayout.id, fragment)
             fragmentTransaction.commit()
             binding.button.visibility = View.GONE
+            binding.weAreHiring.visibility = View.GONE
+        }
+
+        binding.weAreHiring.setOnClickListener {
+            val fragmentManager: FragmentManager = supportFragmentManager
+            val fragmentTransaction: FragmentTransaction = fragmentManager.beginTransaction()
+            val fragment = WeAreHiringFragment()
+            fragmentTransaction.replace(binding.frameLayout.id, fragment)
+            fragmentTransaction.commit()
+            binding.button.visibility = View.GONE
+            binding.weAreHiring.visibility = View.GONE
         }
         setContentView(R.layout.activity_main)
```

Comment:
Ta linijka kodu re-inicjalizuje widok więc podpięte `onClick` nie działają. 
@forceindia712 Możesz naprawić to w tym PR?